system-tests/e2e/import_new_case.cy.js

```diff
@@ -54,4 +54,4 @@ describe('import_new_case.cy.js', () => {
     cy.get('[data-test="confirm-button"]').click();
     cy.get('.modal-container').should('not.exist');
   });
-});
+});
```

Comment:
To reduce the number of changes listed in this commit, let's revert these files to match the ones in the main branch that have no functional changes within them.